core/execution/future/market.go

```diff
@@ -2889,6 +3112,23 @@ func (m *Market) AmendOrder(
 	return m.AmendOrderWithIDGenerator(ctx, orderAmendment, party, idgen)
 }
 
+// handleIsolatedMarginInsufficientOrderMargin stops all party orders
+// Upon position/order update if there are insufficient funds in the order margin, all open orders are stopped and margin re-evaluated.
+func (m *Market) handleIsolatedMarginInsufficientOrderMargin(ctx context.Context, party string) {
+	orders := m.matching.GetOrdersPerParty(party)
+	for _, o := range orders {
+		if !o.IsFinished() {
+			m.unregisterAndReject(ctx, o, types.OrderErrorIsolatedMarginCheckFailed)
+			m.matching.RemoveOrder(o.ID)
```

Comment:
Seems like RemoveOrder set the order status to parked, which is unlikely to be what we want here?
Also we may want to send order events?

Reply:
interesting, because the line above is setting the order to stopped and feature tests are validating that the orders are in state stopped. I suppose swapping the lines may work as desired.